src/common/options/mon.yaml.in

```diff
@@ -27,6 +27,11 @@ options:
   default: 4_K
   services:
   - mon
+- name: mon_max_pool_pg_num
+  type: uint
+  level: advanced
+  default: 64_K
+  fmt_desc: The maximum number of placement groups per pool.
```

Comment:
services?

Reply:
@neha-ojha when an option is ingested, the service property is automatically updated. see https://github.com/ceph/ceph/blob/7fc07dd728b4ee9f83f0e293a5a5ee32020fc7cd/src/common/options/build_options.cc#L40

so unless a given option is shared across multiple services or applications, i think probably we don't need to specify the service explicitly. in this very case, `mon_max_pool_pg_num` does not have `service` specified, so i didn't add it when extracting it from `global.yaml.in` into `mon.yaml.in`.

does this make sense?

Reply:
Yes, thanks for the explanation. Those were missing services, while others were not, so it caught my eye.